code/modules/scavstation_shared_content/_scavs.dme

```diff
@@ -11,9 +11,21 @@
 #include "items/suit_cycler.dm"
 #include "items/turfs.dm"
 #include "map/culture.dm"
+#include "map/culturebaxxid.dm"
+#include "map/languagebaxxid.dm"
 #include "map/overrides.dm"
 #include "lobby/music.dm"
 #include "species/baxxid.dm"
 #include "species/yinglet.dm"
 #include "species/yinglet_organs.dm"
 #include "species/yinglet_accessories.dm"
+
+// BEGIN_INTERNALS
```

Comment:
This is autogenerated but should be removed (the entire block, but most importantly the `#define FILE_DIR .`.

Reply:
Removed autogenerated content in _scavs.dme (line 23 to end of file)